cabal.project

```diff
@@ -71,3 +71,11 @@ if impl (ghc >= 9.12)
 -- IMPORTANT
 -- Do NOT add more source-repository-package stanzas here unless they are strictly
 -- temporary! Please read the section in CONTRIBUTING about updating dependencies.
+
+source-repository-package
```

Comment:
to be removed prior to merge, after https://github.com/IntersectMBO/cardano-cli/pull/1230 is merged and new cardano-cli released